website/content/docs/connect/config-entries/ingress-gateway.mdx

```diff
@@ -1138,6 +1138,45 @@ You can specify the following parameters to configure ingress gateway configurat
                   "Set this configuration to `true` to enable built-in TLS for this listener.<br><br>If TLS is enabled, then each host defined in the `hosts` field will be added as a DNSSAN to the gateway's x509 certificate. Note that even hosts from other listeners with TLS disabled will be added.",
               },
             },
+            {
+              name: 'TLSMinVersion',
+              type: 'string: ""',
+              description: "Set the minimum TLS version supported for this listener. One of `TLS_AUTO`, `TLSv1_0`, `TLSv1_1`, `TLSv1_2`, or `TLSv1_3`.",
+            },
+            {
+              name: 'TLSMaxVersion',
+              type: 'string: ""',
+              description: {
+                hcl:
+                  "Set the maximum TLS version supported for this listener. Must be greater than or equal to `TLSMinVersion`. One of `TLS_AUTO`, `TLSv1_0`, `TLSv1_1`, `TLSv1_2`, or `TLSv1_3`." ,
+                yaml:
+                  "Set the maximum TLS version supported for this listener. Must be greater than or equal to `tls_min_version`. One of `TLS_AUTO`, `TLSv1_0`, `TLSv1_1`, `TLSv1_2`, or `TLSv1_3`." ,
+              },
+            },
+            {
+              name: 'CipherSuites',
+              type: 'array<string>: <optional>',
+              description: `A list of TLS cipher suites to support when negotiating
+                            connections using TLS 1.2 or earlier. If unspecified,
+                            Envoy will use a
+                            [default server cipher list](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto#envoy-v3-api-field-extensions-transport-sockets-tls-v3-tlsparameters-cipher-suites").
+                            <br><br>
+                            Supported values:<br>
+                            \`\`\`TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256 <br>
+                            TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256<br>
+                            TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256<br>
+                            TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256<br>
+                            TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA<br>
+                            TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA<br>
+                            TLS_RSA_WITH_AES_128_GCM_SHA256<br>
+                            TLS_RSA_WITH_AES_128_CBC_SHA<br>
+                            TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384<br>
+                            TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384<br>
+                            TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA<br>
+                            TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA<br>
+                            TLS_RSA_WITH_AES_256_GCM_SHA384<br>
+                            TLS_RSA_WITH_AES_256_CBC_SHA\`\`\``
```

Comment:
```suggestion
                            [default server cipher list](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto#envoy-v3-api-field-extensions-transport-sockets-tls-v3-tlsparameters-cipher-suites"). The list of supported cipher suites can seen in [`types/tls.go`](https://github.com/hashicorp/consul/blob/main/types/tls.go#L154-L169) and is dependent on underlying support in Envoy - future releases of Envoy may remove currently-supported but insecure cipher suites, and future releases of Consul may add new supported cipher suites if any are added to Envoy.
```

Reply:
Linking to the actual implementation source instead could help ensure this content remains current, and is similar to how the Agent TLS config docs handle this https://www.consul.io/docs/agent/options#tls_cipher_suites